src/main/java/frc/robot/HardwareMap.java

```diff
@@ -20,172 +20,188 @@
-            driveController = new XboxController(0);
-            operatorController = new XboxController(1);
-        }
-    }
-
-    /**
-     * Constructs the drive motors, turning motors and turning encoders for each module
-     */
-    private class SwerveModuleHardware {
-        public CANSparkMax frontLeftDriveMotor;
-        public CANSparkMax frontRightDriveMotor;
-        public CANSparkMax backLeftDriveMotor;
-        public CANSparkMax backRightDriveMotor;
-
-        public CANSparkMax frontLeftTurningMotor;
-        public CANSparkMax frontRightTurningMotor;
-        public CANSparkMax backLeftTurningMotor;
-        public CANSparkMax backRightTurningMotor;
-
-        public AbsoluteEncoder frontLeftTurningEncoder;
-        public AbsoluteEncoder frontRightTurningEncoder;
-        public AbsoluteEncoder backLeftTurningEncoder;
-        public AbsoluteEncoder backRightTurningEncoder;
-
-        public SwerveModuleHardware() {
-            frontLeftDriveMotor = new CANSparkMax(Constants.SwervePorts.FRONT_LEFT_DRIVE_MOTOR_PORT,
-                    MotorType.kBrushless);
-            frontRightDriveMotor = new CANSparkMax(Constants.SwervePorts.FRONT_RIGHT_DRIVE_MOTOR_PORT,
-                    MotorType.kBrushless);
-            backLeftDriveMotor = new CANSparkMax(Constants.SwervePorts.REAR_LEFT_DRIVE_MOTOR_PORT,
-                    MotorType.kBrushless);
-            backRightDriveMotor = new CANSparkMax(Constants.SwervePorts.REAR_RIGHT_DRIVE_MOTOR_PORT,
-                    MotorType.kBrushless);
-
-            frontLeftTurningMotor = new CANSparkMax(Constants.SwervePorts.FRONT_LEFT_TURNING_MOTOR_PORT,
-                    MotorType.kBrushless);
-            frontRightTurningMotor = new CANSparkMax(Constants.SwervePorts.FRONT_RIGHT_TURNING_MOTOR_PORT,
-                    MotorType.kBrushless);
-            backLeftTurningMotor = new CANSparkMax(Constants.SwervePorts.REAR_LEFT_TURNING_MOTOR_PORT,
-                    MotorType.kBrushless);
-            backRightTurningMotor = new CANSparkMax(Constants.SwervePorts.REAR_RIGHT_TURNING_MOTOR_PORT,
-                    MotorType.kBrushless);
-
-            frontLeftTurningEncoder = new AbsoluteEncoder(Constants.SwervePorts.FRONT_LEFT_TURNING_ENCODER_PORT, true,
-                    Constants.SwerveConstants.FRONT_LEFT_ROTATION_OFFSET);
-            frontRightTurningEncoder = new AbsoluteEncoder(Constants.SwervePorts.FRONT_RIGHT_TURNING_ENCODER_PORT, true,
-                    Constants.SwerveConstants.FRONT_RIGHT_ROTATION_OFFSET);
-            backLeftTurningEncoder = new AbsoluteEncoder(Constants.SwervePorts.REAR_LEFT_TURNING_ENCODER_PORT, true,
-                    Constants.SwerveConstants.BACK_LEFT_ROTATION_OFFSET);
-            backRightTurningEncoder = new AbsoluteEncoder(Constants.SwervePorts.REAR_RIGHT_TURNING_ENCODER_PORT, true,
-                    Constants.SwerveConstants.BACK_RIGHT_ROTATION_OFFSET);
+                // public OperatorBoard operatorBoard;
```

Comment:
no commented out code

Reply:
That will change once we implement it - in this case the commented out code is fine, it's essentially a reminder that it will be implemented.